src/specfem2D/compute_add_sources_viscoelastic.f90

```diff
@@ -120,45 +124,64 @@ subroutine compute_add_sources_viscoelastic_moving_source(accel_elastic,it,i_sta
   integer :: i_source,i,j,iglob,ispec
   real(kind=CUSTOM_REAL) :: stf_used
   double precision :: hlagrange
-  double precision :: xminSource,vSource,timeval,t_used
+  double precision :: xsrc,zsrc,timeval,t_used
   ! single source array
   real(kind=CUSTOM_REAL), dimension(NDIM,NGLLX,NGLLZ) :: sourcearray
 
   ! checks if anything to do
   if (.not. SOURCE_IS_MOVING) return
 
-  !xminSource = -5000.0d0 !m
-  !vSource = 2150.0d0 !1425.0d0 !1250.0 !m/s
-  xminSource = -60.0d0 !m
-  vSource = 60.0d0 !m/s
-
   if (time_stepping_scheme == 1) then
     ! Newmark
     timeval = (it-1)*deltat
   else
-    call exit_MPI(myrank,'Not implemented!')
+    call exit_MPI(myrank,'Only Newmark time scheme is implemented for moving sources (3)')
   endif
 
-  ! moves and re-locates sources along x-axis
+  if ((myrank == 0) .and. (it < 5)) then
+    do i_source = 1,NSOURCES
+      write(IMAIN,*) ''
+      write(IMAIN,*) 'Your are using moving source capabilities. Please cite:'
```

Comment:
again a citation for each source and the first few time steps, starts filling up the log... :)